app/javascript/src/components/Dashboard/Notes/NewNotePane.jsx

```diff
@@ -1,16 +1,121 @@
 
-import NewNoteForm from "./NewNoteForm";
+import { ROLES } from "./constants";
 
-export default function NewNotePane({ fetchNotes, showPane, setShowPane }) {
+export default function NewNotePane({ setNotes, showPane, setShowPane }) {
   const onClose = () => setShowPane(false);
+  const handleSubmit = values => {
+    setNotes(notes => {
+      const newNote = {
+        id: notes.length + 1,
+        title: values.title,
+        description: values.description,
+        tags: values.tags.map(({ label }) => label),
+        contact: values.tags.map(({ label }) => label)
+      };
+      return [...notes, newNote];
+    });
+    Toastr.success("Note added successfully");
+    onClose();
+  };
   return (
-    <Pane title="Add New Note" isOpen={showPane} onClose={onClose}>
-      <div className="px-6">
-        <NewNoteForm onClose={onClose} refetch={fetchNotes} />
-      </div>
+    <Pane isOpen={showPane} onClose={onClose}>
+      <Pane.Header>
+        <Typography style="h2" weight="semibold">
+          Add New Note
+        </Typography>
+      </Pane.Header>
+      <Formik
+        initialValues={{
+          title: "",
+          description: ""
+        }}
+        onSubmit={handleSubmit}
+        validationSchema={yup.object({
+          title: yup.string().required("Title is required"),
+          description: yup.string().required("Description is required")
+        })}
+      >
+        {({ isSubmitting }) => (
+          <Form>
+            <Pane.Body>
+              <div className="w-full">
+                <Input
+                  className=""
+                  label="Title"
+                  name="title"
+                  placeholder="Enter Title"
+                  required
+                  size="small"
+                />
+                <Input
+                  className="pt-6"
+                  label="Description"
+                  name="description"
+                  placeholder="Enter note description"
+                  required
+                  size="large"
+                />
+                <Select
+                  className="pt-6"
+                  isClearable
+                  isSearchable
+                  label="Assigned Contact"
+                  name="contact"
+                  options={[
+                    {
+                      label: "Value One",
+                      value: "value1"
+                    },
+                    {
+                      label: "Value Two",
+                      value: "value2"
+                    }
+                  ]}
+                  placeholder="Select Role"
+                  required
+                  size="small"
+                />
+                <Select
+                  className="pt-6"
+                  isMulti
+                  label="Tags"
+                  name="tags"
+                  options={ROLES}
+                  placeholder="Select Role"
+                  required
+                  size="small"
+                />
```

Comment:
Please use `space-y-*` utils from tailwindCSS instead of adding `mb-6` to each of the blocks.